data/association-flag-map.yaml

```diff
@@ -109,3 +109,12 @@ columns:
     - name: base_PixelFlags_flag_injected_templateCenter
       bit: 33
       doc: Fake source template injection center in source footprint
+    - name: ext_trailedSources_Naive_flag_off_image
+      bit: 34
+      doc: Trail extends off image
+    - name: ext_trailedSources_Naive_flag_nan
+      bit: 35
+      doc: Trail start or end pixels contain nan
+    - name: ext_trailedSources_Naive_flag_suspect_long_trail
+      bit: 36
+      doc: Trail longer than three psf radii
```

Comment:
Is it necessary to add each of these as formal flags? It looks like you are only ever considering them in aggregate, so I wonder if we can save time (and bits) by having a single "long and/or edgy trailed source" flag and be done with it. Or are you envisioning handling combinations of these flags differently in different circumstances?

Reply:
Not applicable anymore